UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/Layer.cs

```diff
@@ -101,30 +101,33 @@ public void InitFromMatrix()
 		if (matrixMove != null)
 		{
 			Loggy.LogTraceFormat("{0} layer initializing from matrix", Category.Matrix, matrixMove);
-			matrixMove.MatrixMoveEvents.OnRotate.AddListener(OnRotate);
-			//initialize from current rotation
-			OnRotate(MatrixRotationInfo.FromInitialRotation(matrixMove, NetworkSide.Client, RotationEvent.Register));
+			//TODO?
```

Comment:
your todo is to remove this block of commented out code and actually explain what is there to do